pylmod/membership.py

```diff
@@ -28,6 +28,103 @@ def __init__(
         if cuuid is not None:
             self.course_id = self.get_course_id(cuuid)
 
+    def get_group(self, uuid=None):
+        """Get group data based on uuid.
+
+        Args:
+            uuid (str): optional uuid, e.g. '/project/mitxdemosite'
+
+        Raises:
+            PyLmodUnexpectedData: No data was returned.
+            requests.RequestException: Exception connection error
+
+        Returns:
+            dict: group json
+
+        """
+        if uuid is None:
+            uuid = self.cuuid # so does course_id == group_id?
+        group_data = self.get('group', params={'uuid':uuid})
+        return group_data
+
+    def get_group_id(self, uuid=None):
+        """Get group id based on uuid.
+
+        Args:
+            uuid (str): optional uuid, e.g. '/project/mitxdemosite'
+
+        Raises:
+            PyLmodUnexpectedData: No group data was returned.
+            requests.RequestException: Exception connection error
+
+        Returns:
+            int: numeric group id
+
+        """
+        group_data = self.get_group(uuid)
+        try:
+            return group_data['response']['docs'][0]['id']
+        except KeyError, IndexError:
+            failure_message = ('Error in get_group response data - '
+                               'got {0}'.format(group_data))
+            log.exception(failure_message)
+            raise PyLmodUnexpectedData(failure_message)
+        return None
+
+    def get_membership(self, uuid=None):
+        """Get membership data based on uuid.
+
+        Args:
+            uuid (str): optional uuid, e.g. '/project/mitxdemosite'
+
+        Raises:
+            PyLmodUnexpectedData: No data was returned.
+            requests.RequestException: Exception connection error
+
+        Returns:
+            dict: membership json
+
+        """
+        group_id = self.get_group_id(uuid=uuid)
+        uri = 'group/{group_id}/member'
+        mbr_data = self.get(uri.format(group_id=group_id), params=None)
```

Comment:
we generally spell out rather than abbreviate variable names.  So ``mbr_data`` would better be something like ``membership_data``.